### ISSUE TITLE
Reporting missing a11y metadata discouraged by Daisy

### ISSUE THREAD

**Issue by LaZay** [daisy/ace#363]
Two _schema.org_ accessibility properties have been discouraged by Daisy : **schema:accessibilityAPI** and **schema:accessibilityControl**.
http://kb.daisy.org/publishing/docs/metadata/schema.org/accessibilityAPI
http://kb.daisy.org/publishing/docs/metadata/schema.org/accessibilityControl

In Ace report though, they are still listed as missing, when they are not embedded in the EPUB OPF. 

Our EPUB subcontractors are asked to produce natively accessible EPUBs without any missing accessibility metadata in the Ace report.
We need to control our EPUB automatically by using JSON Ace report (i.e. _Violations_ and _Metadata_ at least, as they are the only parts relevant for fully automatic check).
We might need to expose _Violations_ and _Metadata_ Ace control to our potential customers for distribution-sale needs (i.e. requirement of EPUB accessibility details). Our customers might lose confidence in the reality of an PEUB accessibility, if some deprecated metadata were reported as missing by Ace report. They might avoid to take any the risk in buying such an EPUB (though it is OK).

In Ace report, would it possible to:
- either add a 3rd category of metadata = Deprecated/Discouraged, 
- or stop reporting these two metadata as missing?

Thanks!

**Comment by danielweck** [comment]
Fixed via https://github.com/daisy/ace/commit/79948834cc3916f1eda0ff6ee8b923081277142a

**Comment by LaZay** [comment]
Hummm, hoping we did not decide too fast about this. :-/
I realized that this two metadata could be usefull. To be discussed with Daisy probably.
[https://github.com/w3c/a11y-discov-vocab/issues/58](https://github.com/w3c/a11y-discov-vocab/issues/58)
